nip47/controllers/list_transactions_controller.go

```diff
@@ -67,13 +68,13 @@ func (controller *nip47Controller) HandleListTransactionsEvent(ctx context.Conte
 		return
 	}
 
-	transactions := []models.Transaction{}
+	txs := []models.Transaction{}
```

Comment:
can you undo this change? the transactions service should not know about nip47.

Reply:
I did this for this reason: https://github.com/getAlby/hub/pull/834#discussion_r1901552774

Is there any other way around it?

Reply:
I undid the change, supported versions are fetched from cipher package now